package.json

```diff
@@ -167,6 +167,8 @@
     "luxon": "2.0.1",
     "markdown-it": "12.1.0",
     "markdown-table": "2.0.0",
+    "mdast-util-find-and-replace": "1.0.0",
+    "mdast-util-to-string": "1.0.0",
```

Comment:
Same versions as for `remark-github`

Reply:
`remark-github` seems to use `^1.0.0` which not points to some higher version, check lockfile changes below